src/index.css

```diff
@@ -131,6 +131,13 @@ body {
   content: '';
 }
 
+.infinite-scroll-component {
```

Comment:
Maybe there's a way to override the styles of `InfiniteScroll` without CSS? Maybe we can pass `className` or at least `style` into the component?

Reply:
Removed that style from `index.css`.